lib/objects/_layout.scss

```diff
@@ -9,7 +9,7 @@
 /**
  * The default gutter
  */
-$supple-layout-gutter: $supple-space !default;
+$supple-layout-gutter: $supple-space-base !default;
```

Comment:
shouldnt we name this variable `$supple-layout-gutter-base` now? 
I'm not sure if this does something with backwards compatibility.

Reply:
That is a good question, I think we probably should. I will add the variable just like in the first comment.